src/main/java/co/cask/hydrator/salesforce/plugin/BaseSalesforceConfig.java

```diff
@@ -57,6 +59,7 @@
   @Macro
   private final String loginUrl;
 
+  @Nullable
```

Comment:
I am not sure why this became nullable.
Plus note in my PR  I have refactored to use enum instead of constants: https://github.com/data-integrations/salesforce/pull/9/commits/cedeb7d8a0ea8d764e9224334a3d547cfee3c396